### PR TITLE
Add support to install all CHost's errata

### PR DESCRIPTION
**Pull request by swadeley**
Hello

I need this for a new pagination test. Test results will be in that test soon.

Thank you

### CODE REVIEW

**Comment by swadeley** [comment]
Hello

This is the test:

[ Test chost errata pagination #8173 ](https://github.com/SatelliteQE/robottelo/pull/8173)

File: airgun/entities/contenthost.py [hunk 1]
```diff
@@ -141,6 +141,33 @@ def install_errata(self, entity_name, errata_id, install_via=None):
...
+        """Install all errata on a content host
+
+        :param name: content host name to apply errata on
+        :param str install_via: via which mean to install errata. Available
+            options: "katello", "rex", "rex_customize"
+
+        :return: Returns a dict containing task status details
+        """
+        view = self.navigate_to(self, 'Edit', entity_name=entity_name)
+        breakpoint()
```
Comment (lhellebr) [inline airgun/entities/contenthost.py:154]: I suppose this isn't intended?

Comment (swadeley) [reply airgun/entities/contenthost.py:154]: oh dear

Comment (swadeley) [reply airgun/entities/contenthost.py]: @lhellebr Fixed now, thank you for spotting that.

**Review by mirekdlugosz — COMMENTED** [review]
Since this is almost identical to `install_errata` method, I think it would be better to teach existing method to handle special errata id `All`, similar to what we already do in [airgun/entities/host.py](https://github.com/SatelliteQE/airgun/blob/master/airgun/entities/host.py#L336) for entities list:

```python
    def install_errata(self, entity_name, errata_id, install_via=None):
        view = self.navigate_to(self, 'Edit', entity_name=entity_name)
        if errata_id == "All":
            view.errata.select_all.fill(True)
        else:
            view.errata.search(errata_id)
            view.errata.table.row(id=errata_id)[0].widget.fill(True)
        install_via_dict = {
            # skipping rest of method
```

Reviewers: this PR depends on change introduced in #532

**Review by mirekdlugosz — APPROVED** [review]
Thanks, @swadeley !
ACK

**Review by omkarkhatavkar — APPROVED** [review]
ACK

**Pull request merged**